pkg/policy/store.go

```diff
@@ -61,5 +72,105 @@ func (ps *PolicyStore) GetNextPolicy(currentPolicy *v1alpha1.Policy) (*v1alpha1.
 		}
 	}
 
-	return nil, fmt.Errorf("no next policy found")
+	return nil, NO_NEXT_POLICY_FOUND_ERR
+}
+
+func (ps *PolicyStore) GetNextPolicyByName(name string) (*v1alpha1.Policy, error) {
+	log.Println("Identifying next policy to ", name)
+	currentPolicy, err := ps.GetPolicyByName(name)
+	if err != nil {
+		return nil, err
+	}
+
+	policies, err2 := ps.GetSortedPolicies()
+	if err2 != nil {
+		log.Println("Error when fetching policies.")
+		return nil, err2
+	}
+
+	for i, policy := range policies.Items {
+		if policy.Name == currentPolicy.Name {
+			if i+1 < len(policies.Items) {
+				return &policies.Items[i+1], nil
+			}
+			break
+		}
+	}
+
+	return nil, NO_NEXT_POLICY_FOUND_ERR
+}
+
```

Comment:
Please add test cases for the new functions.

Reply:
Ack. That's in the plan.